source/adios2/engine/mixer/HDFMixerWriter.cpp

```diff
@@ -124,7 +124,7 @@ void HDFVDSWriter::AddVar(const VariableBase &var, hid_t h5Type)
     size_t all_counts[m_NumSubFiles][nDims];
 
     //
-    std::vector<hsize_t> dimsf, start, one, count;
+    std::vector<size_t> dimsf, start, one, count;
```

Comment:
Changing this is likely problematic, since those are used in actual HDF5 API calls below.

Probably needs a copy later on instead.